examples/keras_recipes/melanoma_classification_with_tfrecords.py

```diff
@@ -0,0 +1,336 @@
+"""
+Title: How to train a Keras model on TFRecord files
+Author: Amy MiHyun Jang
+Date created: 2020/07/29
+Last modified: 2020/08/06
+Description: Loading TFRecords for computer vision models
+"""
+"""
+## Introduction + Set Up
+
+TFRecords store a sequence of binary records, read linearly. They are useful format for
+storing data because they can be read efficiently. We'll explore how we can easily load
+in TFRecords for our melanoma classifier.
+"""
+
+"""shell
+! pip install gcsfs -q
+"""
+
+import re
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+from functools import partial
+import matplotlib.pyplot as plt
+
+try:
+    tpu = tf.distribute.cluster_resolver.TPUClusterResolver()
+    print("Device:", tpu.master())
+    tf.config.experimental_connect_to_cluster(tpu)
+    tf.tpu.experimental.initialize_tpu_system(tpu)
+    strategy = tf.distribute.experimental.TPUStrategy(tpu)
+except:
+    strategy = tf.distribute.get_strategy()
+print("Number of replicas:", strategy.num_replicas_in_sync)
+
+"""
+Running the following cell will save time in loading the data as it allows for parallel
+processing. TPU can only work with Google Cloud files and cannot be run with local files.
+We want a bigger batch size as our data is not balanced.
+
+"""
+
+AUTOTUNE = tf.data.experimental.AUTOTUNE
+GCS_PATH = "gs://kds-f809fea39df606e89617cd8d8e4cacc083fb6e176982f0e5d69215a8"
+BATCH_SIZE = 16 * strategy.num_replicas_in_sync
+IMAGE_SIZE = [1024, 1024]
+
+"""
+## Load the data
+
+Now we will load in our data. For this notebook, we will be importing the TFRecord files.
+It is good practice to divide the training set data into two. The smaller dataset will be
+the validation set. Having a validation set is useful to slow down overfitting.
```

Comment:
"Having a validation set is useful to slow down overfitting." -> having a validation set is critical whatever you do, so you can remove this sentence